applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,90 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+It is a way to optimize power consumption by adjusting the frequency and voltage of the core according to the current system state.
```

Comment:
I think this sentence might not be necessary and is a bit redundant (half of the info is in the sentence before - what the DFVS does, half in the sentence below - the reason why we use it so power consumption)